rever/main.xsh

```diff
@@ -65,11 +65,13 @@ def compute_activities_completed():
         if 'activity' not in entry:
             continue
         act = entry['activity']
-        if entry['category'] == 'activity-end':
+        if (entry['category'] == 'activity-end' and
+            entry['version'] == $VERSION):
             if act not in acts_done:
                 acts_done[act] = defaultdict(int)
             acts_done[act][entry['data']['start_rev']] += 1
-        elif entry['category'] == 'activity-undo':
+        elif (entry['category'] == 'activity-undo' and
+              entry['version'] == $VERSION):
```

Comment:
This looks good, but a test would be nice :)

Reply:
Does the double conda count? or do you want one where we try to walk back the wrong version?

Reply:
double conda is a fine integration test.  It would be nice if there was a unit test for this, though, so we could more easily identify any underlying issues.  Probably should go in `test_main.py`

Reply:
After that, I think this would be good to go!